public/style/main.css

```diff
@@ -40,6 +68,11 @@ button {
 .btn-savealt {
   background-color : #1fa67a;
 }
+
+.input-group-btn {
+  font-size: 1em;
```

Comment:
you should never need a class to set something to 1em - well not never but this needs to be reevaluated and has to do with the chaotic nature of titles presently